sdk/python/packages/flet/src/flet/controls/material/expansion_panel.py

```diff
@@ -20,23 +20,6 @@ class ExpansionPanel(LayoutControl, AdaptiveControl):
     """
     A material expansion panel. It can either be expanded or collapsed. Its body is
     only visible when it is expanded.
-
-    ```python
-    ft.ExpansionPanelList(
-        width=400,
-        controls=[
-            ft.ExpansionPanel(
-                header=ft.Text("Shipping address"),
-                content=ft.Text("123 Market Street, Springfield"),
-                expanded=True,
-            ),
-            ft.ExpansionPanel(
-                header=ft.Text("Billing address"),
-                content=ft.Text("Same as shipping"),
-            ),
-        ],
-    )
-    ```
     """
```

Comment:
We are including a simple (incomplete) code example for a control to give user understanding of "how much code" (or "what do I need to write") should be written to get what they see on a control screenshot. See here example test: https://github.com/flet-dev/flet/blob/main/sdk/python/packages/flet/integration_tests/examples/material/test_expansion_panel.py
And it's nice to have it in IDE doc popup too.